.build.yml

```diff
@@ -264,7 +268,7 @@ before_scripts:
 build_scripts:
   - if [ ${DISTRO_NAME} == "debian" ];then
   -     export CFLAGS+=" -Wsign-compare -Wunused-parameter"
-  -     cppcheck --enable=warning,style,performance,portability,information,missingInclude .
+  -     cppcheck --enable=warning,performance,portability,information --suppress=missingInclude .
```

Comment:
Why do you drop the style check? This change sneakily does two things at once without explaining.

Reply:
Fixed in additional commits.